contracts/vesting/src/vesting/account/vesting_account.rs

```diff
@@ -0,0 +1,206 @@
+use crate::contract::NUM_VESTING_PERIODS;
+use crate::errors::ContractError;
+use crate::traits::VestingAccount;
+use config::defaults::DENOM;
+use cosmwasm_std::{Coin, Env, Order, Storage, Timestamp, Uint128};
+
+use super::Account;
+
+impl VestingAccount for Account {
+    fn locked_coins(
+        &self,
+        block_time: Option<Timestamp>,
+        env: &Env,
+        storage: &dyn Storage,
+    ) -> Result<Coin, ContractError> {
+        // Returns 0 in case of underflow.
```

Comment:
But should underflow be EVER possible?

Reply:
Thats a good point, no it should never be possible